CAIPs/caip-104.md

```diff
@@ -0,0 +1,229 @@
+---
+caip: 104
+title: Namespace Reference Purpose and Guidelines
+status: Active
+type: Meta
+author: Juan Caballero (@bumblefudge), Wayne Chang (@wyc)
+created: 2022-03-27
+---
+
+# What is a Namespace Reference?
+
+Previously, the specifics of blockchain ecosystems and namespaces were defined
+in single stand-alone CAIPs alongside cross-chain CAIPs. Now, as the system has
+matured and expanded, it has become clear that namespaces are more complex and
+multi-dimensional than can be efficiently governed over time in the current CAIP
+single-document structure; instead, a Namespace Reference is proposed that
+splits out the application of each cross-chain CAIP to a given namespace as a
+distinct document with its own ratification/supersession lifecycle.  This way,
+long-term specifications for low-level primitives like Chain IDs and addresses
+can be defined once and not be superseded by the addition or modification of
+higher-level specifications for RPC interfaces or multiple asset types.
+
+In practical terms, that means each blockchain ecosystem's namespace is defined
+by a folder containing one document per applicable CAIP.  At a high-level, the
+important facts about a namespace, its governance, and its nomenclature can be
+defined/introduced in a `README.md` file, and each CAIP can be defined in a
+`caip{X}.md` file, where X is the number of the ratified CAIP.
```

Comment:
this already sounds like there will be multiple documents in this folder and contradicts the *one* in the beginning